### ISSUE TITLE
Cron unschedule event error for hook: git-updater_tracker_send_event

### ISSUE THREAD

**Issue by therealgilles** [afragen/git-updater#1014]
Started to get this message recently:
```
Cron unschedule event error for hook: git-updater_tracker_send_event, Error code: could_not_set, Error message: The cron event list could not be saved., Data: {"schedule":"weekly","args":[],"interval":604800}
```

**Comment by therealgilles** [comment]
Despite the event recurrence set to 1 week, this event runs every time I try to do: `wp cron event run --due-now`
which means it's not getting properly executed.

```
| git-updater_tracker_send_event                 | 2022-11-04 04:13:12 | now                   | 1 week        |
```

**Comment by afragen** [comment]
Are you in a local environment?

**Comment by therealgilles** [comment]
Not a local environment, no. I also cannot see any settings to allow/deny tracking.

**Comment by afragen** [comment]
As I've worked towards monetizing Git Updater I've added analytics. Opting out is not using the plugin. 

https://github.com/afragen/git-updater/tree/master#analytics

**Comment by afragen** [comment]
I think I've figured out why the cron task continually gets reset. Fix coming.

**Comment by afragen** [comment]
Should be fixed in 11.1.10
